### PR TITLE
Allow serving static assets from `baseDir`

### LINKED ISSUE #65: Esbuild and fastify-swagger-ui build output can't access the swagger-ui-dist static files.
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the bug has not already been reported

### Fastify version

4.17.0

### Plugin version

1.8.1

### Node.js version

18.16.0

### Operating system

macOS

### Operating system version (i.e. 20.04, 11.3, 10)

12.6.5 

### Description

Attempting to run code built with esbuild, fastify, fastify-swagger-ui results in the error:
```
node:internal/modules/cjs/loader:1075
  const err = new Error(message);
              ^

Error: Cannot find module './static/logo.svg'
Require stack:
- /Users/david/workspace/fastifySwaggerTest/dist/index.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1075:15)
    at Function.resolve (node:internal/modules/cjs/helpers:116:19)
    at NE (file:///Users/david/workspace/fastifySwaggerTest/dist/index.js:533:3585)
    at lo.exec (file:///Users/david/workspace/fastifySwaggerTest/dist/index.js:17:1196)
    at Et.om (file:///Users/david/workspace/fastifySwaggerTest/dist/index.js:17:2907)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/Users/david/workspace/fastifySwaggerTest/dist/index.js' ]
}
```

Supplying a valid logo will run the server, but the assets served at static will 404.  

In the provided esbuild.config.mjs, we are defining __filename and __dirname for integration with commonjs libraries.  

### Steps to Reproduce

[Clone repo](https://github.com/drcampbell/fastifyEsbuildError)

run `yarn start` and observe that it doesn't error
run `yarn buildandrun` and observe that it errors

### Expected Behavior

_No response_

### PR DESCRIPTION
**Pull request by davidjbng**
Resolves #65 

This PR documents how to serve static swagger-ui files from a custom directory and fixes serving static assets when the `baseDir` option is set.

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Review by Fdawgs — CHANGES_REQUESTED** [review]
Few issues with doc changes.
Will flesh this out once I'm finished at work.

**Comment by mcollina** [comment]
@Fdawgs ptal

File: README.md [hunk 1]
```diff
@@ -319,6 +319,51 @@ fastify.register('@fastify/swagger-ui', {
 
 Note that this behavior is disabled by default in `@fastify/swagger-ui`.
 
+### Bundling
+
+If you want to bundle Swagger UI with your application, you need to copy the swagger-ui static files to your server yourself and set the `baseDir` option to point to your folder.
+
```
Comment (Fdawgs) [inline README.md:325]:
```suggestion
To bundle Swagger UI with your application, the swagger-ui static files need to be copied to the server and the `baseDir` option set to point to the file directory.

```

**Review by Fdawgs — CHANGES_REQUESTED** [review]

**Review by Fdawgs — APPROVED** [review]

**Comment by DavidTanner** [comment]
Any idea on when this can be merged?

**Pull request merged**